handler/deployment/deployment.go

```diff
@@ -202,7 +202,7 @@ func (d *Deployment) deploymentSuccess(ctx context.Context, correlationID, evalu
 	for {
 		select {
 		case <-ctx.Done():
-			log.InfoC(correlationID, "bailing on deployment status", minLogData)
+			log.Event(ctx, "bailing on deployment status", log.INFO, minLogData)
```

Comment:
looks like we are no longer logging `correlationID` here, wonder if it would be best to add it to `minLogData` earlier

(and `logData` later, too)

Reply:
As above, I think the `correlationID` is actually the `msg.ID` which is in the context so it will be logged out as the trace id.